libs/pages/elewa/home/src/lib/components/home-jobs-section/home-jobs-section.component.html

```diff
@@ -1,8 +1,7 @@
-<section class="home-jobs-section">
-    <span >
-        Become part of the family
-    </span>
-    <span >
-        View <a class="homeLink" routerLink="/jobs">open opportunities.</a>
-    </span>
-</section>
\ No newline at end of file
+<elewa-group-reusable-home-jobs-section
```

Comment:
The issue description required the component to be called "elewa-call-to-action-banner"

Check description again